### PR TITLE
Import hotfix

### PR DESCRIPTION
**Pull request by nathan-miller23**
* Fixed really subtle circular import error with `human_aware_rl` that caused race conditions with tensorflow imorts
  * Moved harl imports to top of file, added some extra dependencies to Dockerfile
  * Still don't quite understand what caused it but it appears to be working now

* Added some dummy agents (ones that stay and perform random actions) as well as an rllib agent that was trained on overcooked for ~5 training iters that will hopefully server as examples for users
* Added detailed instructions on the agent training and loading into demo pipeline I've been using in the README

### CODE REVIEW

**Comment by nathan-miller23** [comment]
Probably want to leave this open until 1-2 *real* agents are trained and added into the pre-trained agents directory

**Comment by micahcarroll** [comment]
Thanks for doing this!

What do you mean by "real" agents by the way?

**Review by micahcarroll — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -48,9 +48,9 @@ The default branch for both repos is currently `master`.
 
 ## Using Pre-trained Agents
 
-Overcooked-Demo can dynamically load pre-trained agents provided by the user. In order to use a pre-trained agent, a pickle file should be added to the `agents` directory. The final structure will look like `static/assets/agents/<agent_name>/agent.pickle`
+Overcooked-Demo can dynamically load pre-trained agents provided by the user. In order to use a pre-trained agent, a pickle file should be added to the `agents` directory. The final structure will look like `static/assets/agents/<agent_name>/agent.pickle`. Note, to use the pre-defined rllib loading routine, the agent directory name must start with 'rllib', and contain the appropriate rllib checkpoint, config, and metadata files. For more detailed info and instructions see the [RllibDummy_CrampedRoom](server/static/assets/agents/RllibDummy/) example agent.
```
Comment (mesutyang97) [inline README.md:51]: Just one minor link correction here:

For more detailed info and instructions see the [RllibDummy_CrampedRoom](server/static/assets/agents/RllibDummy_CrampedRoom/)

Comment (nathan-miller23) [reply README.md]: Good catch! I updated it

**Comment by nathan-miller23** [comment]
> What do you mean by "real" agents by the way?

I meant agents that were trained for more than a few timesteps. I tried to train an agent on `counter_circuit` but the potential function in `master` of harl is different than the one I have in my `nathan_dev` branch, and thus my hyperparameters didn't transfer over well. I believe @mesutyang97 trained an agent on `cramped_room`, in which case I think just adding that one successfully trained agent is sufficient. If you have any comments or think we should include additional trained agents please let me know

**Comment by micahcarroll** [comment]
Ok cool, yes this sounds like a good idea. Initially thought you had ended up training the agents yourself (till completion). But yes, let's wait for @mesutyang97 to add the actual agent(s) in this PR and then merge it?

**Comment by mesutyang97** [comment]
Yes, I have trained some good PPO agents for `cramped_room` on my local machine. I am working to get `overcooked_demo` installed so that I can test them out before including them in this PR. Should be done soon.

**Comment by mesutyang97** [comment]
The good PPO Rllib self-play agent for `cramped_room` has been uploaded, and I can verify locally that it is working as it should. @micahcarroll Please perform a final check before we merge this into master

**Comment by micahcarroll** [comment]
@mesutyang97 You should probably add to the agent README the exact command you used to train this agent in the human_aware_rl repo. Later on we should clean this up and store this information in a consistent manner, but for now this seems like the most obvious choice to give people something to work off of.

**Pull request merged**